src/shared/constants/index.js

```diff
@@ -0,0 +1,7 @@
+export const genresList = [
```

Comment:
is that really constant? it's mock data, please, take a look at the screenshot I've provided